pages/contribute.md

```diff
@@ -6,44 +6,58 @@ main_nav_id: contribute
+If you want to contribute to the standard library, follow [the standard process for contributing](https://github.com/JetBrains/kotlin/blob/master/docs/contributing.md).
 
-### Presentations
+If you want to contribute to a kotlinx library, follow the contribution process described for each library, such as [kotlinx.coroutines](https://github.com/Kotlin/kotlinx.coroutines),
+ [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization/blob/master/CONTRIBUTING.md), and [ktor](https://github.com/ktorio/ktor/blob/master/CONTRIBUTING.md).
 
-If you've given or are giving presentations on Kotlin, let us know. We'll [feature it on the list](http://kotlinlang.org/docs/events.html)
+If you have a library that could be useful for others, let us know via <feedback@kotlinlang.org>.
 
-### Kotlin-x Projects and Community Projects
+## Contribute to the documentation
 
-Kotlin ships with a very small runtime and it's the aim to keep it that way. We believe other functionality that is missing from the standard runtime library can be developed
-as Kotlin Contributions under the Kotlin-X umbrella or as individual projects by community members. If you have a library that you think
-others might find useful, let us know. If you want to contribute to any of the existing ones, check them out.
+If you have helpful information for [Kotlin documentation](https://kotlinlang.org/docs/reference/) or have found a documentation issue, 
+feel free to check out [the documentation source code on GitHub](https://github.com/JetBrains/kotlin-web-site/tree/master/pages/docs)
+and send us a Pull Request. Please stay in touch with us because we may have questions and comments on your changes. Otherwise, we won't be able to incorporate them. 
 
-### Translations
+The documentation site is built using Markdown and Jekyll. 
 
-You are welcome to translate the Kotlin documentation into your own language and to publish your translation on your Web site.
-However, note that we will not be able to host your translation in this repository and to publish it on kotlinlang.org.
-This site is the official documentation for the language, and we need to be able to ensure that all the information here
-is correct and up-to-date. Doing this with a language that no one on the team speaks is unfortunately not feasible at this time.
+## Create tutorials or videos
 
-### Spread the word
+If you've created tutorials or videos for Kotlin, please share them with us via <feedback@kotlinlang.org>. We would be happy to add them to [Awesome Kotlin](https://kotlin.link/) maintained by the community.
 
-Kotlin is a new language but we have high hopes for it and we need people that believe in it to spread the word! 
+## Translate documentation to other languages
 
+You are welcome to translate the Kotlin documentation into your own language and publish your translation on your web site.
```

Comment:
You are welcome to translate the Kotlin documentation into your own language and publish the translation on your website.

Reply:
done